### PR TITLE
Add log level setting in allocator

### LINKED ISSUE #1879: Adding allocator log level
**Is your feature request related to a problem? Please describe.**
I want to suppress the allocator logs.

**Describe the solution you'd like**
Add a setting to helm.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by 8398a7**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup
> /kind documentation

/kind feature

> /kind hotfix

**What this PR does / Why we need it**:
When the number of requests is high, most of the logs become noise.
I wanted to set the log level to `error`, so I put out this PR.

**Which issue(s) this PR fixes**:

Closes #1879

**Special notes for your reviewer**:

### CODE REVIEW

File: cmd/allocator/main.go [hunk 1]
```diff
@@ -66,10 +67,19 @@ const (
 func main() {
 	conf := parseEnvFlags()
 
-	logger.WithField("version", pkg.Version).WithField("ctlConf", conf).
+	logger.WithField("version", pkg.Version).WithField("conf", conf).
```
Comment (8398a7) [inline cmd/allocator/main.go:70]: I think it's a remnant of the controller.
I fixed it, but I'll put it back if it looks problematic.

Comment (markmandel) [reply cmd/allocator/main.go:70]: Yeah, can we roll this back. Better to be consistent between each binary.

**Review by markmandel — COMMENTED** [review]
This looks great - just that one small reversion, and this is good to go!

**Comment by 8398a7** [comment]
Thank you. I put it back.

> Better to be consistent between each binary.

Just out of curiosity, what are the benefits of consistency between binaries?

**Comment by markmandel** [comment]
> Just out of curiosity, what are the benefits of consistency between binaries?

- People know that they can find the same thing in each log, so there's no mystery on where to find them
- If people are parsing the structured log format, there's no breaking changes there
- Developers know what the standard is across the project, so there's less ambiguity.

**Comment by 8398a7** [comment]
It's certainly a break change.
I learned a lot about the others as well. Thank you.

**Review by markmandel — APPROVED** [review]
Great stuff!

**Pull request merged**